### PR TITLE
LWDEV-9330 Remove obsolete balance related functionality functionality

### PR DESCRIPTION
**Pull request by rocketmannot**

### CODE REVIEW

File: src/main/kotlin/com/lykke/matching/engine/balance/WalletOperationsProcessor.kt [hunk 1]
```diff
@@ -11,24 +11,26 @@ import com.lykke.matching.engine.database.common.entity.PersistenceData
...
 import org.apache.log4j.Logger
 import java.math.BigDecimal
 import java.util.Date
-import java.util.concurrent.BlockingQueue
+import kotlin.collections.HashMap
+import kotlin.collections.List
+import kotlin.collections.forEach
+import kotlin.collections.getOrPut
+import kotlin.collections.isNotEmpty
+import kotlin.collections.toList
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/matching/engine/balance/WalletOperationsProcessor.kt]: redundant imports

File: src/main/kotlin/com/lykke/matching/engine/messages/MessageType.kt [hunk 2]
```diff
@@ -1,20 +1,13 @@
 package com.lykke.matching.engine.messages
 
-import java.util.HashMap
+import java.util.*
 
 enum class MessageType (val type: Byte){
     RESPONSE(0)
     ,PING(1)
-    ,CASH_OPERATION(2)
     ,OLD_BALANCE_UPDATE(6)
```
Comment (barsukovskii) [inline src/main/kotlin/com/lykke/matching/engine/messages/MessageType.kt]: Remove also this value

File: src/main/kotlin/com/lykke/matching/engine/balance/WalletOperationsProcessor.kt (hunk 1 above)
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/balance/WalletOperationsProcessor.kt]: fixed

File: src/main/kotlin/com/lykke/matching/engine/messages/MessageType.kt (hunk 2 above)
Comment (rocketmannot) [reply src/main/kotlin/com/lykke/matching/engine/messages/MessageType.kt]: fixed

**Pull request merged**